Makefile

```diff
@@ -91,12 +98,15 @@ test-e2e: ## Run the e2e tests
 ###################
 # Running Targets #
 ###################
-.PHONY: load-image deploy teardown run run-local run-e2e run-e2e-local
+.PHONY: load-image load-bundle deploy teardown run run-local run-e2e run-e2e-local
 
 IMAGE_LOAD_COMMAND=kind load docker-image
 load-image: ## Load-image loads the currently constructed image onto the cluster
 	$(IMAGE_LOAD_COMMAND) $(IMAGE)
 
+load-bundle: ## Loads the currently constructed Combo bundle onto the cluster
```

Comment:
hmm, this target is very similar to the one above, I wonder if it adds enough value to include it? Or we can simply advise to `make load-image IMAGE=<bundle-image>` instead?

Reply:
Yeah I think as long as we document it well that's totally reasonable.